catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/metacard/RegistryObjectMetacardType.java

```diff
@@ -103,31 +84,21 @@ public RegistryObjectMetacardType(String name, Set<AttributeDescriptor> descript
     }
 
     private void addRegistryAttributes() {
-        descriptors.addAll(BasicTypes.BASIC_METACARD.getAttributeDescriptors());
-        addQueryableBoolean(REGISTRY_IDENTITY_NODE, false);
-        addQueryableBoolean(REGISTRY_LOCAL_NODE, false);
-        addQueryableDate(DATA_END_DATE);
-        addQueryableDate(DATA_START_DATE);
-        addQueryableDate(LAST_PUBLISHED);
-        addQueryableDate(LIVE_DATE);
-        addQueryableString(DATA_SOURCES, true);
-        addQueryableString(DATA_TYPES, true);
-        addQueryableString(ENTRY_TYPE, false);  //objectType
+        descriptors.add(BasicTypes.BASIC_METACARD.getAttributeDescriptor(Metacard.POINT_OF_CONTACT));
+        addQueryableString(SECURITY_LEVEL, true); //securityLevel
```

Comment:
All of the attribute adds were previously alphabetized. There's are advantages to that, keeping types grouped (booleans; dates; strings) and then strict ordering within. It makes changes easier to review.